network/sites.csv

```diff
@@ -757,6 +757,8 @@ CSWB,10,-41.57143,174.12668,268,,WGS84,External GPS Device,2013-07-23T05:20:10Z,
 CSWC,10,-41.57146,174.12672,269,,WGS84,Internal GPS Clock,2013-07-24T05:05:00Z,2013-11-19T10:00:00Z
 CSWS,20,-41.57143,174.12668,268,,WGS84,External GPS Device,2013-07-23T05:20:10Z,2013-11-19T10:00:00Z
 CTND,10,-43.47686,172.21852,170,,WGS84,External GPS Device,2010-09-05T01:35:02Z,2010-09-29T23:00:00Z
+CTPS,20,-41.278513,174.785939,5,,WGS84,Internal GPS Clock,2024-09-23T00:00:00Z,9999-01-01T00:00:00Z
+CTPS,21,-41.278513,174.785939,5,,WGS84,Internal GPS Clock,2024-09-23T00:00:00Z,9999-01-01T00:00:00Z
```

Comment:
double checking  the elevation, was it mentioned in the ticket as well?

Reply:
I copied the coordinates and elevation from the locality in the network PR for this site.